.rubocop.yml

```diff
@@ -46,7 +46,7 @@ RSpec/ContextWording:
     - given
 
 RSpec/ExampleLength:
-  Max: 30
+  Max: 37
```

Comment:
It stop making sense if you keep increasing this. I recommend revisit the test by breaking them down.